qa/c8-orchestration-cluster-e2e-test-suite/package.json

```diff
@@ -5,6 +5,7 @@
   "scripts": {
     "test": "playwright test",
     "test:local": "node --env-file=.env ./runTest.js",
+    "test:tasklist": "playwright test tests/tasklist --project=chromium",
```

Comment:
@esraagamal6 Please open a follow up PR adding a Playwright project like in [here](https://github.com/camunda/camunda/blob/main/tasklist/client/playwright.config.ts#L43-L47) on Tasklist

This we have each test suite properly configured for each component and can call just `playwright test --project=tasklist` which each project config encapsulated.

Reply:
Looks good